### PR TITLE
B's PR

### PR DESCRIPTION
**Pull request by bharathmr**
PR for description

### CODE REVIEW

**Review by bharathmr — COMMENTED** [review]
minor change

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# tutorialrepo
\ No newline at end of file
+# tutorialrepo
+
+Repo for tutorial
```
Comment (willmathias) [inline README.md:3]: really wanna change?

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# tutorialrepo
\ No newline at end of file
+# tutorialrepo
```
Comment (willmathias) [inline README.md:1]: shud we  rename?

**Review by willmathias — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (bharathmr) [reply README.md:1]: let's leave it as-is for now

File: README.md (hunk 1 above)
Comment (bharathmr) [reply README.md:3]: yes, please

**Review by willmathias — APPROVED** [review]
Good to go!!

**Comment by willmathias** [comment]
Lets reopen

**Pull request merged**